src/plugins/external/session.ts

```diff
@@ -0,0 +1,30 @@
+import fastifySession from '@fastify/session'
+import fp from 'fastify-plugin'
+import { Auth } from '../../schemas/auth.js'
```

Comment:
Path aliasing in the future?

Reply:
Unlike frontend projects, I am not sure it would work well on a node.js server.

Reply:
I have no opinion on the subject if others are ok, why not :+1: